atomicapp/nulecule/main.py

```diff
@@ -102,6 +103,40 @@ def __init__(self, app_spec, destination=None,
         self.answers_file = answers_file
         self._process_answers()
 
+    @staticmethod
```

Comment:
@dustymabe Although I am tempted to break the function into units, but I don't see much value in it as I don't see much reusability for the units.